cmd/service-controller/policies.go

```diff
@@ -125,3 +126,38 @@ func (p *PolicyManager) outgoingLink() http.Handler {
 		}
 	})
 }
+
+func (p *PolicyManager) writeAllowedByInfo(tw *tabwriter.Writer, title string, allowedInfo map[string][]string) {
```

Comment:
Why is this a method on PolicyManager? It doesn't seem to use any members of that struct. If the allowedBy type was exported, this method could perhaps be defined on that?

Even if not, perhaps this could be moved into that package and have a WriteTable() method o the PolicyDebug struct?

Reply:
It is certainly misplaced.
I will move it into the renamed `PolicyInfo` type as a `String()` (stringer implementation).